packages/stencil/components/tsconfig.components.test.json

```diff
@@ -13,6 +13,7 @@
   "include": [
     "accordion/src",
     "cart/src",
+    "checkbox/src",
     "checkbox-button/src",
```

Comment:
- `"checkbox/src",` shoud not be here after rebasing
- remove `checkbox-button/src`